services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -1,20 +1,20 @@
 package cromwell.pipeline.service
 
-import java.nio.file.{ Path, Paths }
-
+import java.nio.file.{Path, Paths}
 import akka.http.scaladsl.model.StatusCodes
 import cromwell.pipeline.datastorage.dao.repository.utils.TestProjectUtils
 import cromwell.pipeline.datastorage.dto.File.UpdateFileRequest
+import cromwell.pipeline.datastorage.dto.PipelineVersion.PipelineVersionException
 import cromwell.pipeline.datastorage.dto._
 import cromwell.pipeline.utils._
-import org.mockito.Matchers.{ any, eq => exact }
+import org.mockito.Matchers.{any, eq => exact}
 import org.mockito.Mockito.when
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatest.{AsyncWordSpec, Matchers}
```

Comment:
Please reformat it properly as it was before

Reply:
Sorry. Forgot to use`Reformat Code`